packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -36,7 +36,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:3.2.8' // renovate.json regexManagers
+    return 'defi/defichain:HEAD-fd79aec14' // renovate.json regexManagers
```

Comment:
can we add a `TODO` here so we wont forget to update

Reply:
Should we update this with a much later version?